lodash.src.js

```diff
@@ -10060,10 +10060,10 @@
     function result(object, path, defaultValue) {
       var result = object == null ? undefined : toObject(object)[path];
       if (result === undefined) {
+        path = toPath(path);
```

Comment:
Hmmm is there a reason to check `!isKey(path, object)`? It's implicitly done via `var result = object == null ? undefined : toObject(object)[path];`.